app/models/user.rb

```diff
@@ -4,4 +4,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   validates :username, presence: true, uniqueness: true
+
+  # Not sure about this line of code. Please, check!
```

Comment:
If we delete a user, do we want their reviews to remain?  
Yes - if someone has spammed the site then deleting the user will remove all of the content they have added.
No - a user can delete themselves (i guess? data protection etc...), but we would not want to lose all of the users contributions....

I think 'No' - lets not have reviews dependent on user destroy.... but what are your thoughts?
If anyone had 'spammed' the site, we would easily be able to remove all reviews by a user manually.